.github/workflows/publish-on-change.yml

```diff
@@ -53,7 +53,7 @@ jobs:
             echo "Workflow triggered by push. Using SHA: $SHA_SHORT"
 
             DOCKER_TAGS="quay.io/inference-perf/inference-perf:${SHA_SHORT},quay.io/inference-perf/inference-perf:latest,quay.io/inference-perf/inference-perf:main"
-            CHART_VERSION="main-${SHA_SHORT}"
+            CHART_VERSION="0.0.0-${SHA_SHORT}"
```

Comment:
Can we not map to the release version we are currently on? For example, if we are at 0.2.0, this can be 0.2.0-sha